markdownlint.js

```diff
@@ -41,7 +41,7 @@ const fsOptions = {encoding: 'utf8'};
 const processCwd = process.cwd();
 
 function readConfiguration(userConfigFile) {
-  const jsConfigFile = /\.js$/i.test(userConfigFile);
+  const jsConfigFile = /\.(c)?js$/i.test(userConfigFile);
```

Comment:
I think the parentheses are unnecessary?

Reply:
You are right. I have a habit of always wrapping with parentheses for any optional regexes. :slightly_smiling_face: